### DISCUSSION TITLE
No info when requesting for order address

### DISCUSSION THREAD

**Discussion by ivancortesromero** [amzn/selling-partner-api-models#4432] [General]
I have a public app with the [Inventory and Order Tracking](https://developer-docs.amazon.com/sp-api/docs/inventory-and-order-tracking-role) role, which should allow me to request the order’s address (via the _getOrderAddress_ operation). However, when I call this endpoint, the response only includes the order ID without any additional information, such as the address.

![Screenshot 2024-12-16 at 1 06 15 PM](https://github.com/user-attachments/assets/18024df1-98d5-4b96-a530-c00dd9c64fd7)

**Comment by Tong8s** [comment]
Hi Iván,

Order shipping address are Personally Identifiable Information (PII), you'll need to get restricted role and use restricted data token (RDT) to get this information through getOrderAddress API:

Prerequisite for calling getOrderAddress: https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-use-case-guide#prerequisites, see "To access buyer and shipping address information, you must:" section
Authorization with restricted data token: https://developer-docs.amazon.com/sp-api/docs/authorization-with-the-restricted-data-token
